operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/informer/ManagedInformerEventSource.java

```diff
@@ -13,15 +18,31 @@
     extends CachingEventSource<R, P>
     implements ResourceEventHandler<R> {
 
+  @SuppressWarnings("rawtypes")
+  private static final ConcurrentMap<ResourceConfigurationAsKey, InformerManager> managedInformers =
```

Comment:
TBH, don't know what to think about this static context. 
What is this magic number 17?

Reply:
This will be started and stopped multiple times. Don't like the facts that it is even shared between the controllers. It will be started and stopped multiple times.

Reply:
Well, you're the one who wanted to share informers between sources… :)
We now avoid starting or stopping multiple times anyway.

Reply:
Pls add integration test for this, like when 2 reconcilers added with 2 Informers, so there is only one informer in the background.